certora/LiquityStrategy_verification_report.md

```diff
@@ -0,0 +1,204 @@
+![Certora](https://hackmd.io/_uploads/H1yqrfBZY.png)
+# Formal Verification of LiquityStrategy contract
+
+## Summary
+
+This document describes the specification and verification of LiquityStrategy using the Certora Prover. 
+
+The scope of this verification is the [`LiquityStrategy.sol`](https://github.com/lindy-labs/sc_solidity-contracts/blob/abe1005872ea3d49052d4ede3b643fb14d4f8542/contracts/strategy/liquity/LiquityStrategy.sol) contract. Its specification is available [here](specs/LiquityStrategy.spec).
+
+The Certora Prover proved the implementation of the LiquityStrategy contract is correct with respect to formal specifications written by the security team of Lindy Labs.  The team also performed a manual audit of these contracts.
+
+## List of Issues Discovered
+
+**Severity: <span style="color:blue">Medium  </span>**
+
+| Issue:            | Possible failure to withdraw from Vault | 
+| --------          | -------- |    
+| Description:      | If the vault calls the `withdrawToVault(uint256 amount)` function on the strategy to withdraw an amount greater than the LUSD in the SP but smaller than that amount + the value held in ETH, the vault may receive less LUSD than the amount, hence when the vault transfer the amount of LUSD to the user, the transaction will fail due to insufficient balance. | 
```

Comment:
I am not the right person to validate this part, ok fei?!

Reply:
Right, don't worry about this part. This issue is raised. Solgryn is aware of the issue.